### DISCUSSION TITLE
Adding boxes around taskbar items

### DISCUSSION THREAD

**Discussion by kittenShark42** [valinet/ExplorerPatcher#1577] [General]
Pretty new here (and a bit lost)!  Windows 11 with Open Shell and Explorer Patcher installed; works pretty well but I am trying to get the "boxes" around the taskbar items; the old way like this:

![image](https://github.com/valinet/ExplorerPatcher/assets/137316997/edf4af0e-de25-4df9-8078-bb264fa5edf9)

I tried retrobar, but due to my display the text looked blurry no matter what I trid but  I also didn't like I couldn't change the start icon - SCT did not run well at all, and taskbow had similar issues.

I quite enjoy Explorer Patcher though, but is there any simple way that I can add those boxes?

Thanks!

**Comment by Yakov5776** [comment]
The current version of ExplorerPatcher doesn't have a built-in option for adding boxes around taskbar items like the classic Windows appearance, your best bets are to try using other third-party tools to achieve this.

*(P.S. If you're well-equipped on implementing such functionality, please consider making a pull request!)*

Thanks,
Yakov
